### DISCUSSION TITLE
Add photo to album from detail view

### DISCUSSION THREAD

**Discussion by Trekky12** [ente-io/ente#3291] [Enhancements]
It would be great to have the possibility to add a photo to a album while in the detail view of the photo especially on desktop. Currently this is only possible from the album view.

The use case is selecting the best photos of a album. In the album view the photos are not fully shown and a selection of the best photos is not that easy.

**Comment by RolfSievert** [comment]
I think this is important on the mobile app as well. If opening an image through the memories feature (e.g. 2 years ago), there is no way to add/remove that image to/from an album.

For me it does not have to be in the details view, although it would be practical since the album it's in is listed there. Alternatively it could be an option alongside "edit", "share", etc when viewing the image.

**Comment by danieldangl** [comment]
Yes please, I moved over from Google Photos and it didn't even came to my mind that this might not be possible.

I used to all the time look at photos in detail view (both mobile and desktop) and add them to albums - often even multiple, to group them for events, printing, (later) sharing in other apps and so on.
Easily accessible tagging would work for that too, but since albums already exist I'd assume adding it to the detail view would be faster to implement.

**Reply by vishnukvmd** [reply]
Yes, this is on our radar. We will make this happen soon.

**Comment by Trekky12** [comment]
I think this can be closed because it is implemented for mobile in https://github.com/ente-io/ente/pull/7174 and for desktop/web in https://github.com/ente-io/ente/pull/7962

Thank you very much 🥳
